lib/core/stream.nit

```diff
@@ -417,6 +417,9 @@ abstract class Writer
 	# Write a single byte
 	fun write_byte(value: Byte) is abstract
 
+	# Writes a single char
+	fun write_char(c: Char) do write(c.to_s)
```

Comment:
Yep, nothing better to date.

Though I guess the to_s chars can be precomputed in a table and compiled along with the rest, this should improve the runtime of all the `to_s` on `Char`, making them virtually free.

Maybe later, anyhow, with #2006 there will be a proper `write_char` implementation, therefore this should not be an issue anymore :)